guidelines/2016-03-08-github-guidelines.md

```diff
@@ -0,0 +1,116 @@
+
+
+### Issues
+
+The first steps before creating an issue of any type on the repository is to search through the repository's Open/Closed issues to see whether there's a ticket that replicates the same issue you are having and if there is a fix in progress or has been fixed (alternatively you can use GitHub's search feature for this). If there is no same issue that you are experiencing, then open a new ticket on the affected repository.
+
+#### Reporting bugs
+
+Found a bug with Origami? Fill out the template below before submitting an issue. Due to a recent new addition to GitHub, the template can be included automatically when an issue is opened.
+
+##### The template for reporting bugs
+
+```
+[Short description of the problem]
+
+**Steps taken to reproduce the bug**
+1.
+2.
+3.
+...
+
+**Expected outcome**
+[Describe the expected outcome]
+
+**Actual outcome**
+[Describe the actual outcome]
+
+Note: You can include screenshots/animated gifs if it will help to fix this bug in this section.
+
+**What software/processes was used to replicate this issue**
+1. [Browser(s) (include the version number(s))]
+2. [Operating System(s)]
+3. [Which Origami modules were affected (and include the version number)]
+4. [Did you build manually or use the build service?]
+5. [If you build manually then could you say which version of Origami Build Tools & node were used?]
+
+* The problem started happening recently, it did not happen in the older versions of the module: [Yes/No]
+* The problem can be easily reproduced every time and does not happen randomly: [Yes/No]
+```
+
+#### Suggesting improvements to Origami
+
+Before opening an issue which is for enhancement purposes, look through all the Open/Closed issues in the repository to see if there is not a similar suggestion to what you would be suggesting to improve Origami.
+
+```
+[Short description of the enhancement suggestion]
+
+**Steps which could explain how the improvement would work with Origami**
+1.
+2.
+3.
+...
+
+**Current outcome and suggested improvement's outcome**
+
+[Explain the current outcome and the suggested improvement outcome]
+
+**Why would this improvement be more useful than the current version?**
+
+[Explain why the suggested improvement would be better than the current version]
+
+Note: You can include screenshots/animated gifs if you think it will help the cause for the suggested improvement.
+```
+
+
+### Pull requests
+
+Before committing a pull request, please make sure that it covers the criteria.
+
+- Follows the [Origami best practices](/docs/developer-guide/general-best-practices/).
+- Include screenshots/animated gifs wherever possible.
+
+
+#### Commit messages
+
+- Use present tense when committing i.e. `Add module-name` not `Added module-name`.
+- The first line of the commit message to be 72 characters or less.
+- Add references to issues or pull requests in the description with a short explanation of the changes proposed in the pull request.
```

Comment:
(proposed addition)
- Explain *why* you've made a change as well as what the change is